config/default.yml

```
@@ -4,6 +4,11 @@ Minitest:
     - '**/test/**/*'
     - '**/*_test.rb'
 
+Minitest/AnyAssertions:
+  Description: 'This cop checks for at least one assertion (or flunk) in tests.'
+  Enabled: 'pending'
```

> I guess this is where `rubocop:disable` comes in. It doesn't feel like great practice to me to have no assertions inside your test.

I think this cop is worth adding to RuboCop Minitest 👍  On the other hand, assuming `rubocop:disable` pragma is not better for user experience. So, I have the same concerns as @andyw8's opinion. Can you set the default to `false`?

```suggestion
  Enabled: false
```